<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lessons_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>901</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Уроки</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="structure_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>59</y>
      <width>891</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Материал всех уроков структурирован по различным эпохам для упрощения понимания и изучения сложных исторических процессов</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="choose_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>881</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Выберите эпоху:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="epochesbox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Древний мир (до 500г н.э)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Средневековье (500-1500гг)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Эпоха Возрождения и Реформации (1500-1700гг)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Новое время (1700-1900гг)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>XX век (1900-2000гг)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Современный мир (2000г - настоящее время)</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="lessons_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>281</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Перейти к урокам по данной эпохе</string>
    </property>
   </widget>
   <widget class="QPushButton" name="return_button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Вернуться назад</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
